Workaround for the following problem:
rm -f ru.gmo && /usr/bin/msgfmt -c --statistics -o ru.gmo ru.po
ru.po:21:9: invalid multibyte sequence
ru.po:21:10: invalid multibyte sequence
ru.po:21:11: invalid multibyte sequence
ru.po:21:12: invalid multibyte sequence
ru.po:21:13: invalid multibyte sequence
ru.po:21:14: invalid multibyte sequence
ru.po:21:15: invalid multibyte sequence
ru.po:21:16: invalid multibyte sequence
ru.po:21:17: invalid multibyte sequence
ru.po:21:18: invalid multibyte sequence
ru.po:21:19: invalid multibyte sequence
ru.po:21:20: invalid multibyte sequence
ru.po:21:22: invalid multibyte sequence
ru.po:21:23: invalid multibyte sequence
ru.po:21:24: invalid multibyte sequence
ru.po:21:25: invalid multibyte sequence
ru.po:21:26: invalid multibyte sequence
ru.po:21:27: invalid multibyte sequence
ru.po:21:29: invalid multibyte sequence
ru.po:21:30: invalid multibyte sequence
/usr/bin/msgfmt: too many errors, aborting
make[2]: *** [ru.gmo] Error 1
make[2]: Leaving directory `/var/tmp/fst/src/bash-3.2/po'
make[1]: *** [stamp-po] Error 2
make[1]: Leaving directory `/var/tmp/fst/src/bash-3.2/po'
make: [install] Error 2 (ignored)
diff -Naur bash-3.2.orig/po/LINGUAS bash-3.2/po/LINGUAS
--- bash-3.2.orig/po/LINGUAS	2006-01-10 23:51:33.000000000 +0100
+++ bash-3.2/po/LINGUAS	2006-10-14 00:44:27.000000000 +0200
@@ -1,2 +1,2 @@
 # Set of available languages.
-en@quot en@boldquot ru
+en@quot en@boldquot
